Gemfile

```diff
@@ -12,8 +12,8 @@ gem 'rails', '~> 7.0.4'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
-
+# gem 'sqlite3', '~> 1.4'
```

Comment:
If you're not gonna use sqlite3 anymore it's perfectly fine to get rid of it, no need to keep this line commented